### ISSUE TITLE
[CrashReport 2017-07-01 10:56:29] After detaching debugger, removing breakpoints fails

### ISSUE THREAD

**Issue by X39** [ArmA-Studio/ArmA.Studio#53]
```<?xml version="1.0" encoding="utf-8"?>
<root>
  <version>0.3.6390.34477</version>
  <report><![CDATA[clicked the debugger detach button (red square) and tried to remove a breakepoint]]></report>
  <stacktrace><![CDATA[Unterbrochene Pipe.
   bei System.IO.Pipes.PipeStream.CheckWriteOperations()
   bei System.IO.Pipes.PipeStream.Write(Byte[] buffer, Int32 offset, Int32 count)
   bei Dedbugger.DebuggerCore.WriteMessage(JsonNode node) in C:\projects\arma-studio\Dedbugger\DebuggerCore.cs:Zeile 270.
   bei Dedbugger.DebuggerCore.RemoveBreakpoint(BreakpointInfo b) in C:\projects\arma-studio\Dedbugger\DebuggerCore.cs:Zeile 292.
   bei ArmA.Studio.DebuggerContext.BreakpointManager_OnBreakPointsChanged(Object sender, BreakPointsChangedEventArgs e) in C:\projects\arma-studio\ArmA.Studio\DebuggerContext.cs:Zeile 118.
   bei System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   bei ArmA.Studio.BreakpointManager.RemoveBreakpoint(ProjectFile pff, Int32 line) in C:\projects\arma-studio\ArmA.Studio\BreakpointManager.cs:Zeile 99.
   bei ArmA.Studio.BreakpointManager.RemoveBreakpoint(BreakpointInfo bpi) in C:\projects\arma-studio\ArmA.Studio\BreakpointManager.cs:Zeile 89.
   bei ArmA.Studio.UI.BreakPointMargin.OnMouseDown(MouseButtonEventArgs e) in C:\projects\arma-studio\ArmA.Studio\UI\BreakPointMargin.cs:Zeile 122.
   bei System.Windows.UIElement.OnMouseDownThunk(Object sender, MouseButtonEventArgs e)
   bei System.Windows.Input.MouseButtonEventArgs.InvokeEventHandler(Delegate genericHandler, Object genericTarget)
   bei System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   bei System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   bei System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   bei System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   bei System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   bei System.Windows.UIElement.RaiseEvent(RoutedEventArgs args, Boolean trusted)
   bei System.Windows.Input.InputManager.ProcessStagingArea()
   bei System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)
   bei System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   bei System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   bei System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   bei System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   bei MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   bei MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   bei System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   bei System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
]]></stacktrace>
  <trace />
</root>```

**Comment by dedmen** [comment]
Also noticed this when the Debugger side closes the Pipe unexpectedly. Just wrap a try/catch around the Write/Read.
Not sure if Read needs it tho.

**Comment by X39** [comment]
should be fixed by b652b63
